integration_tests/pages/page.ts

```diff
@@ -138,6 +142,21 @@ export default abstract class Page {
     })
   }
 
+  shouldContainRadioButtons(options: Array<{ label: string; value: string }>): void {
```

Comment:
Would this method work with multiple radio input questions on a page? (The details page will have this) Does the containing `.govuk-radios` element need passing in to constrain it?

Perhaps that iteration can come when adding the details page

Reply:
Will address that separately.